custom-elements/custom-elements-registry/define.html

```diff
@@ -249,6 +259,19 @@
       }, `If constructor.prototype.${name} is ${data.name}, should ${data.success ? 'succeed' : 'throw a TypeError'}`);
     });
   });
+
+  // 13.11.1. Let observedAttributesIterable be Get(constructor, "observedAttributes").
+  // Rethrow any exceptions.
+  test(() => {
```

Comment:
This test should not be added; it is no longer valid ever since https://github.com/whatwg/html/pull/1422